source/docs/software/troubleshooting/introduction.rst

```diff
@@ -0,0 +1,83 @@
+Introduction to Troubleshooting

```

Comment:
It's probably worth mentioning keeping a written record of the steps that were taken (why they were taken), and what the observations were.